src/screens/TopicPageScreen/index.js

```diff
@@ -95,7 +94,7 @@ const TopicPageScreen = (props) => {
     : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT;
   const headerShowHeight =
     navigationHeight + dimen.size.TOPIC_FEED_HEADER_HEIGHT + topPosition + normalize(4);
-  const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition;
+  const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems to remove the addition of `topPosition` to `headerHideHeight`. If `topPosition` was previously needed to correctly calculate the height at which the header should hide, this could potentially cause issues. Please make sure this change doesn't affect the UI behavior.
```diff
- const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition;
+ const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->